gqlalchemy/how-to-guides/query-builder/graph-projection.md

```diff
@@ -0,0 +1,61 @@
+---
+id: graph-projection
+title: How to create a graph projection
+sidebar_label: Create a graph projection
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+To learn about graph projections and how they are used to 
+construct subgraphs take a look at [how we implemented the graph projection feature](https://memgraph.com/blog/how-we-designed-and-implemented-graph-projection-feature).
```

Comment:
I think it would be better to use these resrouces

https://memgraph.com/docs/mage/how-to-guides/run-a-subgraph-module
https://memgraph.com/blog/how-we-designed-and-implemented-graph-projection-feature

And you can link them in labels, instead of full sentences. 
You can see an example of a blog label in the Run a subgraph module, and you can see an example of the how-to label in this reference guide -> https://memgraph.com/docs/memgraph/reference-guide/indexing).

So my proposal is to add 2 labels here, for the how to and blog :)